5-Heroku/server/app.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const todoRouter = require('./routers/todoRouter')
+const cors = require('cors')
```

Comment:
don't use cors. in general, you should never write production code that is there not b/c of production requirement. i.e this is here b/c of the webpack dev server and not b/c something else

Reply:
remove this line